tox.ini

```diff
@@ -19,7 +19,7 @@ commands = py.test --collect-only tests
 [testenv:flake8]
 deps =
     flake8
-    flake8-mutable
+#    flake8-mutable
```

Comment:
?

Reply:
By mistake I forgot to remove the comment.  When I running tox locally this module is problematic on my mac and couldn't make it work from some reason. So I have to comment it when running tox locally ;) and sometime when I do git add -u, I add all changed files to commit and that's why it's here like this.  Will fix today ;)

Reply:
done